app/containers/RoomHeader/index.tsx

```diff
@@ -102,6 +120,8 @@ class RoomHeaderContainer extends Component<IRoomHeaderContainerProps, any> {
 			subtitle = subtitleProp;
 		}
 
+		const sourceT = sourceType ? { ...sourceType, connected } : undefined;
```

Comment:
Why are you passing `connect` as a prop here, if in the `OmnichannelRoomIcon` file you are getting this prop with useSelector?

Reply:
You are right! I forgot to remove the prop in this file